web/cobrands/fixmystreet/assets.js

```diff
@@ -989,6 +989,8 @@ max_resolution - either a number, or a hash mapping cobrand to number. This
     provides the maximum resolution at which an asset layer will be displayed
     (and when first shown, the map may zoom in to this level). The hash is for
     the case where a cobrand shows a different map with different resolutions.
+    Run `fixmystreet.map.resolution` in the browser's js console to see the
```

Comment:
This will work fine, though I guess it should be `fixmystreet.map.getResolution()` (which I think handles edge cases where e.g. the base layer isn't present as per the comment at https://github.com/mysociety/ol2/blob/76657c6857748cd795f1e4c26b1228e71fcceaad/lib/OpenLayers/Map.js#L2262)